<launch>

<node pkg="visibility_graph" type="visibility_graph_node" name="visibility_graph_node" output="screen">
    
    <param name="map/polygon_vertices_size" value="4"/>
    <param name="map/frame" value="enu"/>

    <rosparam param="planning/start"> [-4.0, 7.00, 2.75] </rosparam>
    <rosparam param="planning/end"> [15.0, -12.0, 1.00] </rosparam>

    <rosparam param="planning/obstacles"> 
        [2.0, 3.00, 2.00, 7.00, -1.00, 7.00, 0.00, 3.00, 3.00,
        2.0, 0.00, 2.00, -2.00, 0.00, 0.00, 1.00, -2.00, 3.00,
        3.00, 1.00, 2.00, 1.00, 3.00, -1.00, 2.00, -1.00, 4.00,
        5.00, 0.00, 6.00, 0.00, 6.00, 2.00, 5.00, 2.00, 1.00,
        5.00, -2.00, 6.00, -2.00, 6.00, 0.00, 5.00, 0.00, 3.00,
        8.00, -6.00, 9.00, -6.00, 9.00, -1.00, 8.00, -1.00, 4.00,
        4.0, -8.00, 10.00, -8.00, 9.00, -6.00, 4.0, -6.00, 4.00,
        11.00, -6.00, 12.00, -6.00, 12.00, 2.00, 11.00, 2.00, 4.00] 
    </rosparam>

    <param name="planning/protected_zone" value="0.5"/>

</node>

<!-- Launch RViz with the demo configuration -->
<node name="rviz" pkg="rviz" type="rviz"
    args="-d $(find visibility_graph)/rviz/vis_graph.rviz"/>

</launch>